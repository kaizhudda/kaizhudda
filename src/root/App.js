import React from 'react';
import Home from '../views/home';

import './App.scss';

const App = () => {
  return (
    <div className="App row center-xs">
      <div className="col-xs-12">
        Kaiz Hudda App
        <Home />
      </div>
    </div>
  );
}

export default App;
