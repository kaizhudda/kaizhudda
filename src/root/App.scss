body {
  * {
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
  }
  margin-top: 0px;
  margin-bottom: 0px;
}
.App {}