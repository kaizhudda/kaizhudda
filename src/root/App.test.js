import React from 'react';
import { shallow } from 'enzyme';
import App from './index';

let wrapped = shallow(<App />); 

describe('App', () => {
  it('should render the App Component correctly', () => {
    expect(wrapped).toMatchSnapshot();
  });

  it('should contain the Header Component', () => {
    expect(wrapped.find('Header').exists()).toBe(true);
  });

  it('should contain the Routes Component', () => {
    expect(wrapped.find('Routes').exists()).toBe(true);
  });

  it('should contain the React Helmet Component', () => {
    expect(wrapped.find('HelmetWrapper').exists()).toBe(true);
  });
});
