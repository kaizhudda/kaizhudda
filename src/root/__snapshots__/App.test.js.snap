// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App Component correctly 1`] = `
<div
  className="App row center-xs"
>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title>
      Kaiz Hudda - Front End React Developer - Digital Resume
    </title>
    <meta
      content="Hi, I'm Kaiz Hudda, a versatile Front-End React Developer with 3+ years of experience leveraging React, Redux and JavaScript to build commercial web-based applications. Focused on engineering user centric solutions for achieving greater customer satisfaction. Thrive in working in a collaborative agile environment to deliver high quality products."
      name="description"
    />
  </HelmetWrapper>
  <div
    className="col-xs-12"
  >
    <Header />
    <Routes />
  </div>
</div>
`;
